samples/change-notification/csharp/ChangeNotification/Controllers/NotificationController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using ChangeNotification.Helper;
+using ChangeNotification.Helper;
+using ChangeNotification.Model;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ChangeNotification.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class NotificationsController : Controller
+    {
+        private readonly IConfiguration _config;
+        private readonly ILogger _logger;
+        public static  ITurnContext stepContext;
```

Comment:
Storing stepContext is not correct way to send proactive message, please take a look at sample to [send proactive](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/16.proactive-messages/Controllers/NotifyController.cs#L34) message. Please make the necessary changes.